data/data/gcp/network/firewall-control.tf

```diff
@@ -23,6 +23,19 @@ resource "google_compute_firewall" "master_ingress_ssh" {
   target_tags   = ["${var.cluster_id}-master"]
 }
 
+resource "google_compute_firewall" "bootstrap_ingress_ssh" {
```

Comment:
these need to be part of the bootstrap module, so that these are gone with bootstrap node.

Reply:
ack, fixing

Reply:
updated